test/test_foreach.py

```diff
@@ -0,0 +1,55 @@
+import torch
```

Comment:
As this is a new file, it should be added to some run_tests.py to make sure it is run?